graylog2-server/src/main/java/org/graylog2/bootstrap/preflight/web/resources/PreflightStatusResource.java

```diff
@@ -37,11 +37,11 @@
 public class PreflightStatusResource {
 
     private final Version version = Version.CURRENT_CLASSPATH;
-    private final PreflightConfigService preflightConfigService;
+    private final PreflightConfigService preflightConfigServiceIf;
```

Comment:
What does that "If" that ends the name mean?

Reply:
ah, refactoring related, will fix that. Thx!